test/test_restful.py

```diff
@@ -276,6 +276,33 @@ def initthread():
     return True
 
 
+def launch_rust_cloudagent():
+    """Start up the cloud agent"""
+    global agent_process, script_env, FORK_ARGS
+    if agent_process is None:
+        agent_process = subprocess.Popen(["cargo", "run", "--manifest-path", f"{PACKAGE_ROOT}/../rust-keylime/Cargo.toml", "--bin", "keylime_agent"],
```

Comment:
Why do not call directly `keylime_agent`, instead of running it from `cargo run`?

Reply:
Good point - I just didn't know how to call it directly, but it's in the `.cargo/bin` directory. Added.